src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/summarize.py

```diff
@@ -32,6 +33,7 @@ def parse_arguments():
     parser.add_argument("--out", help=out_help, default="")
     parser.add_argument("--output_filename", help=output_filename_help, default="report")
     parser.add_argument("--report_tag", help=report_tag, default="")
+    parser.add_argument("--report_version", help=report_version, default="")
```

Comment:
```suggestion
    parser.add_argument("--report_version", type=str, help=report_version, default="")
```
?

Reply:
Minor change which is not required. Won't fix it according extra CI load